src/Route.jsx

```diff
@@ -13,10 +17,27 @@ const STOP_ICON_MAPPING = {
 
 const BUS_ICON_MAPPING = {
   marker: {
-    x: 0, y: 0, width: 128, height: 128, mask: false,
+    x: 0, y: 0, width: 128, height: 128, mask: true,
   },
 };
 
+/* https://en.wikipedia.org/wiki/Template:MUNI_color */
+const MUNI_COLOR = [
+  { line: 'E', color: [102, 102, 102] },
+  { line: 'F', color: [240, 230, 140] },
+  { line: 'J', color: [250, 166, 52] },
+  { line: 'K', color: [86, 155, 190] },
+  { line: 'L', color: [146, 39, 143] },
+  { line: 'M', color: [0, 135, 82] },
+  { line: 'N', color: [0, 83, 155] },
+  { line: 'S', color: [255, 204, 0] },
+  { line: 'T', color: [211, 18, 69] },
```

Comment:
Since there's only `KT` and no `T`, could we comment this out for now?
```suggestion
  // { line: 'T', color: [211, 18, 69] },
```